config/clients/java/template/build.gradle.mustache

```diff
@@ -39,7 +44,7 @@ ext {
     {{/swagger2AnnotationLibrary}}
     jackson_version = "2.14.1"
     jakarta_annotation_version = "1.3.5"
-    junit_version = "4.13.2"
+    junit_version = "4.13.2" // TODO: JUnit 5 (Jupiter)
```

Comment:
TODO: Upgrade this before investing more in tests.

Reply:
For now I'll just break the integration tests out to their own test suite. That can use JUnit 5 at Gradle's built-in version.

Changing this turned out to have some weird edge cases. It didn't matter what I did to the imports in the mustache files, they'd get rewritten to `import org.junit.<Class>;` instead of whatever I put down. It seems like the OpenAPI generator is doing post-processing to add/change JUnit dependencies and not strictly coming from mustache or mustache includes.

Reply:
Let's discuss this

Reply:
Turns out... the local mustache files were not being used at all. @rhamzeh figured out the issue was related to directory structure, and it's been fixed in [16b7592](https://github.com/openfga/sdk-generator/pull/158/commits/16b7592277ef25f91872346c97a5ab520dd794e8)